exercism/javascript/simple-cipher/simple-cipher.js

```diff
@@ -0,0 +1,138 @@
+const Cipher = function (key) {
```

Comment:
it would be good to add the problem prompt, maybe in another file so it's more readable and a credit link to the problem.